packages/http-client/__tests__/proxy.test.ts

```diff
@@ -145,6 +145,30 @@ describe('proxy', () => {
     expect(bypass).toBeFalsy()
   })
 
+  it('checkBypass returns true if host with subdomain in no_proxy', () => {
+    process.env['no_proxy'] = 'myserver.com'
+    const bypass = pm.checkBypass(new URL('https://sub.myserver.com'))
+    expect(bypass).toBeTruthy()
+  })
+
+  it('checkBypass returns true if host with leading dot in no_proxy', () => {
+    process.env['no_proxy'] = '.myserver.com' 
```

Comment:
I added some whitespace in my last commit to re-trigger the checks, that's probably it since they passed before! I've reverted it, but am not in a position to run the `format` or `lint-fix` right now. I'll take a look at that later tonight

Reply:
Or it works I guess :D 

Want me to run the `npm audit fix` and fix the bad dependencies?

Reply:
Done it, should pass audit now

Reply:
The lining and auditing is fixed now!